apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ir/Argument.kt

```diff
@@ -1,5 +1,8 @@
 package com.apollographql.apollo.compiler.ir
 
+import com.squareup.moshi.JsonClass
+
+@JsonClass(generateAdapter = true)
```

Comment:
The IR is made serializable (to save the Fragment in the metadata). The serialization is only done if `generateMetadata` is true so as not to slow down compilation in the general case.